README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Eugene's Introduction
+***
+Hello! 
+<img align="left" src="https://avatars2.githubusercontent.com/u/56091387?s=400&u=ebca95a0653a33a6df4455dbde6ab52345270761&v=4" width="60" height="60">  
```

Comment:
I am glad that you managed to make the image work, but you are using HTML language instead of markdown (technically they are related).